content/blog/k8s-operator/index.md

```diff
@@ -0,0 +1,98 @@
+---
+path: '/k8s-operator'
+title: 'Kubernetes Operators: 3 reasons to use, 2 reasons not to'
```

Comment:
I would change this, because ending on use sounds a bit weird

- Kubernetes Operators: 3 reasons to use them (and 2 reasons not to)
- 3 Reasons to use Kubernetes Operators (and 2 reasons not to)